tools/downloader/converter.py

```diff
@@ -57,14 +57,21 @@ def prefixed_subprocess(prefix, args):
         return p.wait() == 0
 
 def convert_to_onnx(model, output_dir, args, stdout_prefix):
-    pytorch_converter = Path(__file__).absolute().parent / 'pytorch_to_onnx.py'
+    if model.pytorch_to_onnx_args:
+        converter = Path(__file__).absolute().parent / 'pytorch_to_onnx.py'
+        converter_args = [string.Template(arg).substitute(conv_dir=output_dir / model.subdirectory,
+                                                            dl_dir=args.download_dir / model.subdirectory)
+                                for arg in model.pytorch_to_onnx_args]
+    if model.caffe2_to_onnx_args:
+        converter = Path(__file__).absolute().parent / 'caffe2_to_onnx.py'
```

Comment:
Why not use the `caffe2.python.onnx.bin.conversion` module that's included in Caffe2?